sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/keyvault_pipeline.hpp

```diff
@@ -132,6 +140,23 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Common { n
       return Azure::Response<T>(factoryFn(*response), std::move(response));
     }
 
+    template <class T>
+    Azure::Response<T> SendRequest(
+        Azure::Core::Context const& context,
+        Azure::Core::Http::HttpMethod method,
+        std::function<std::string()> serializeContentFn,
+        std::function<T(Azure::Core::Http::RawResponse const& rawResponse)> factoryFn,
+        std::vector<std::string> const& path)
+    {
+      auto serialContent = serializeContentFn();
+      auto streamContent = Azure::Core::IO::MemoryBodyStream(
+          reinterpret_cast<const uint8_t*>(serialContent.data()), serialContent.size());
+
+      auto request = CreateRequest(method, &streamContent, path);
```

Comment:
unrelated to this PR, but change `CreatedRequest` to accept `BodyStream` as ref rather than pointer.

Reply:
`CreateRequest` need to init a Http Request which takes a BodyStream*